test/cql-pytest/run

```diff
@@ -27,9 +27,14 @@ run.wait_for_services(pid, [
     lambda: run.check_rest_api(ip),
     lambda: check_cql(ip)
 ])
-success = run.run_pytest(sys.path[0], ['--host=' + ip] + sys.argv[1:])
+vnodes_success = run.run_pytest(sys.path[0], ['--host=' + ip, '--replication=vnodes'] + sys.argv[1:])
+tablets_success = run.run_pytest(sys.path[0], ['--host=' + ip, '--replication=tablets'] + sys.argv[1:])
```

Comment:
I don't like this change I don't think test/cql-pytest/run should run all the tests twice and have double the code for tracking passes and failures. I think it's complicates the code and confusing. I think the "run" script should run only one set of tests. If you want to run it once with tablets and once without, run the "run" script twice.

Reply:
What I'm suggesting is that you can have an option to run with tablets, and another to run without tablets - but if you want to run both you'll need to run it twice. Jenkins can be modified to run it twice. Or test.py can be modified to run the same suite twice, with two different parameters.

I'm worried that this is a slippery slope - today you added 2 runs, in 7 months we'll have 2^7 runs with 2^7 combinations of 7 options.

Reply:
Found a much more elegant solution, have a look at v2.

I don't think we will add any other feature, which will require running all tests with/without them, like we do with tablets right now.

Reply:
We had **exactly** the same "with and without" need for schema changes over Raft, and had a "--raft" option in test/cql-pytest.
We'll have exactly the same thing if we ever want to rewrite a major feature - e.g:
1. LWT over raft instead of paxos
2. Strongly consistent tables
3. MV rewrite